### DISCUSSION TITLE
Prompt sometimes appearing in the transcript

### DISCUSSION THREAD

**Discussion by tehSlothman** [openai/whisper#1150] [Q&A]
I've got an initial prompt set to attempt to guide Whisper towards using UK spellings, but occasionally this prompt will appear at the start of the transcript, I think always replacing what should be the transcript of the first 30 seconds of the audio. Sometimes it'll be word for word, sometimes it'll be switched up a bit. Are there any settings that will reduce the likelihood of this happening?

**Comment by Anna-Pinewood** [comment]
I have similar story. I am using initial prompt to set punctuation in transcript. I put a small piece of audio, where there is text, i heard it myself. But with `initial_prompt='Добрый день.'` whisper doesn't show real transcription. However when i change initial prompt to "Итак," it starts working. I don't understand how it works.
I use openai-whisper, version 20231117.
My code:
```
import whisper

model = whisper.load_model("large-v3")
audio = whisper.load_audio(path_audio)
result = model.transcribe(audio, language='ru', 
                          initial_prompt='Добрый день.',
                          verbose=True)
torch.cuda.empty_cache()
# [00:00.000 --> 00:29.980]  Добрый день.

```
But when i change initial_prompt to
 ```
result = model.transcribe(audio, language='ru', 
                          initial_prompt='Итак.',
                          verbose=True)
# [00:00.000 --> 00:05.240]  С которой, я уверена, каждый из вас знаком, возможно, даже пробовали и примерно понимаете, что это.
# [00:05.380 --> 00:09.440]  То есть история развития технологий насчитывает уже достаточно большое количество времени.
# ...
```

**Comment by Purfview** [comment]
> occasionally this prompt will appear at the start of the transcript

Try https://github.com/openai/whisper/pull/1903 bugfix. You can install it like that:

`python -m pip install --upgrade --force-reinstall "whisper-patch-1.zip"` [ downloaded from [https://github.com/Purfview/whisper/tree/patch-1](https://github.com/Purfview/whisper/tree/patch-1) ]

If that alone doesn't help then try `hallucination_silence_threshold=2` option.

**Comment by rahulbansal16** [comment]
It is occurring for me also. I can't put this option in the GROQ.
Does it occur for the turbo model as well?
